### DISCUSSION TITLE
How do I change support density?

### DISCUSSION THREAD

**Discussion by texaslonghorn75** [OrcaSlicer/OrcaSlicer#1543] [Q&A] [answered]
I've recently started using Orca as my go to slicer. I'm coming from using Cura 5.4. I'm having a hard time trying to find how to change the density of supports. On Cura it's easy to find but on Orca not so much. Is it named something else?

**Comment by Glenn1963** [comment, marked as answer]
I'm not 100% sure if it's what you're looking for, but changing the Base Pattern / Base Pattern Spacing might accomplish your goal.

I guess also the Interface settings including Spacing
